File: ujin/1주차_입출력/2439.py
N = int(input())
for i in range(N):
    for k in range(N - (i + 1)):
        print(' ', end='')
    for j in range(i+1):
        print('*', end='')
    print()

#다른 방법
'''
N = int(input())
for i in range(N):
    print(' '*(N-i-1),end='')
    print('*'*(i+1))
'''

File: minto/1추차_입출력/2439.js
let input = Number(require('fs').readFileSync('/dev/stdin').toString());

for (let i = 1; i <= input; i++) {
  let result = '';
  for (let j = input; j > i; j--) {
    result += ' ';
  }
  for (let j = 0; j < i; j++) {
    result += '*';
  }
  console.log(result);
}
